docs/RC/GetInteriorVehicleDataCapabilities/index.md

```diff
@@ -0,0 +1,151 @@
+## RC.GetInteriorVehicleDataCapabilities
+Type
+:	Function
+
+Sender
+:	SDL
+
+Purpose
+:	Obtain vehicle capabilities on available modules controls in available locations
+
+_**Trigger to send:**_   
+RSDL sends _RC.GetInteriorVehicleDataCapabilities_ to the vehicle after receiving a valid _GetInteriorVehicleDataCapabilities_ request from remote-control application.
+ 
+_**Back-up:**_   
+RSDL has a logic to _read the capabilities from the predefined file_ so that to respond to the application in case _the vehicle fails to respond or responds with error_ to _RC.GetInteriorVehicleDataCapabilities_.
+
+To switch this logic on, the OEM must ensure the following:   
+   1.	"InteriorVehicleDataCapabilities.json" file that lists all available locations and available modules must be created (please see "Response" in "JSON message example" below for the expected structure).
+   2.	"InteriorVDCapabilitiesFile" parameter of _SmartDeviceLink.ini_ file must contain the path to this file.   
+               _Example:_ InteriorVDCapabilitiesFile = ./plugins/InteriorVehicleDataCapabilities.json
+
+In case any of the above 2 steps is omitted, RSDL will:   
+•	transfer vehicle's error to the mobile app in case the vehicle responds GetInteriorVehicleDataCapabilities with an error.   
```

Comment:
Transfer the vehicle's error to the mobile app in case the vehicle responds to the GetInteriorVehicleDataCapabilities request with an error.  ***